src/components/Article.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({
+  title,
+  text,
+  date,
+}) => (
+  <>
+    <h1>title</h1>
+    <span>date</span>
+    <p>text</p>
```

Comment:
```suggestion
    <h1>{title}</h1>
    <span>{date}</span>
    <p>{text}</p>
```
Тут должны данные из props отображаться